### ISSUE TITLE
Sibling dependencies cause compile to fail

### ISSUE THREAD

**Issue by ConorPFarrell** [reactiverse/vertx-maven-plugin#167]
We have a project which is set up like this:

main
- common/pom.xml
- backend/(pom.xml, pom-backend.xml)

The backend module, which uses the vertx-maven-plugin, depends on the common module and we have a Reactor build that specifies this. However, when we run mvn compile, we receive the following error:

```
[INFO] --- vertx-maven-plugin:1.0.20:initialize (vmp) @ backend ---
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for backend-parent 1.0.0:
[INFO] 
[INFO] common ............................................. SUCCESS [  2.042 s]
[INFO] backend ............................................ FAILURE [  0.108 s]
[INFO] backend-parent ..................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.367 s
[INFO] Finished at: 2019-10-09T10:46:23+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal io.reactiverse:vertx-maven-plugin:1.0.20:initialize (vmp) on project backend: Unable to copy WebJar dependency, com.company:common has not been resolved
```

However, running mvn package works fine. The error seems to be down to the way the initialize step resolves its dependencies - I'm guessing it doesn't take into account the possibility that the dependency will be provided by a sibling module?

**Comment by tsegismont** [comment]
If you have more info about this one (small reproducer) please reopen

**Comment by fedinskiy** [comment]
For anyone, who found this issue from the search: this was fixed in version 1.0.21, probably by https://github.com/reactiverse/vertx-maven-plugin/pull/146, so try updating the plugin if you see something similar.

**Comment by tsegismont** [comment]
Thank you @fedinskiy for the update
